capirca/aclgen.py

```diff
@@ -46,6 +45,7 @@
 from capirca.lib import naming
 from capirca.lib import nftables
 from capirca.lib import nsxv
+from capirca.lib import nsxt
```

Comment:
The `nsxt.py` module itself, the most important aspect of this PR, is missing from this pull request. Please make sure you include everything required in the pull request.

Reply:
Done